packages/frontend/src/features/sqlRunner/components/BarChartFieldConfiguration.tsx

```diff
@@ -234,15 +286,21 @@ export const BarChartFieldConfiguration = () => {
             </Config>
             <Config>
                 <Config.Section>
-                    <Config.Heading>{`Y-axis`}</Config.Heading>
-
+                    <Config.Group>
+                        <Config.Heading>{`Y-axis`}</Config.Heading>
+                        <AddButton
+                            onClick={() => dispatch(addYAxisField())}
+                        ></AddButton>
+                    </Config.Group>
                     {yLayoutOptions &&
                         yAxisFields &&
-                        yAxisFields.map((field) => (
+                        yAxisFields.map((field, index) => (
                             <YFieldsAxisConfig
-                                key={field.reference}
+                                key={field.reference + index}
```

Comment:
I think this is safe - or we have IDs generated on the fly

Reply:
This is fine